config/sync/.htaccess

```diff
@@ -19,6 +19,6 @@ SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
 </Files>
 
 # If we know how to do it safely, disable the PHP engine entirely.
```

Comment:
Wait, what?  Where did this come from?

Reply:
Hm. Looking into it. 

I wonder if composer update could change his.

Reply:
Although, the change makes sense because we're using php7 now.